### ISSUE TITLE
max_line_length

### ISSUE THREAD

**Issue by mnot** [Mr0grog/editorconfig-textmate#19]
Map max_line_length to TM's Wrap Column.

**Comment by Mr0grog** [comment]
Sorry I’ve been so slow to respond. I haven’t had a lot of time to work on this plug-in lately, so I’m not sure when I might be able to get to it. If you’d like to take a crack at it and submit at PR, that’d be great.

As a side note, I’m not sure mapping the property to the “Wrap Column” is really the right thing to do. There’s a post on the mailing list about exactly this, actually: https://groups.google.com/forum/#!topic/editorconfig/sinFhuirEgI

Basically, mapping to the wrap column/ruler is relatively easy, but doesn’t quite match the description of what the property should do: 
> Forces hard line wrapping after the amount of characters specified (https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length)
